### ISSUE TITLE
DOS games show error, possibly due to  Dosbox being required inside flatpak sandbox

### ISSUE THREAD

**Issue by ell1e** [flathub/io.github.sharkwouter.Minigalaxy#52]
DOS games show an unexpected error for me:

![](https://private-user-images.githubusercontent.com/64124388/296886343-984305bb-1fef-437f-8ff3-4674b5217652.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1NzMxMjUsIm5iZiI6MTcxNzU3MjgyNSwicGF0aCI6Ii82NDEyNDM4OC8yOTY4ODYzNDMtOTg0MzA1YmItMWZlZi00MzdmLThmZjMtNDY3NGI1MjE3NjUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDA3MzM0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMwMmIyYTZhYzc1ZWFhMWE1MDBjZGM3ZTUxMDY2ZGFmZjUzMDg5YTNiYzEwNTZkOWJhNzQ2OGM4OGJkOTE4YmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CjrjzfBvH30RL3EWiLfzV-0zfR1jXbxCdJH9jBZVZfE)

It was suggested [here https://github.com/sharkwouter/minigalaxy/issues/592] that this requires dosbox to be installed. I have that installed, but it seems like the flatpak doesn't pick it up. I assume maybe it needs to be installed inside the flatpak sandbox?

**Comment by ell1e** [comment]
Sadly, all DOS games still seem to be broken :pensive: if there's something I can do to help test a fix, let me know.

**Comment by sharkwouter** [comment]
I think "native" dosbox games should work. The issue is most likely that wine does not run the bundled dosbox very well. We're working on improving the wine installation of games. It could be that this will fix it, but I cannot guarantee it.

**Comment by ell1e** [comment]
Yeah it seems like it's most likely some install issue, with something not in the place where it should be. Some wine dosbox games work, after all.
